react/tic-tac-toe/src/app/index.js

```diff
@@ -1,11 +1,29 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { ConnectedRouter } from 'connected-react-router';
+import { Route, Switch, BrowserRouter, Redirect } from 'react-router-dom';
+
+import { history } from '../redux/store';
 
 import Game from './screens/Game';
+import Login from './screens/Login';
+import AuthRoute from './components/AuthRoute';
+import { loggedIn } from './screens/Login/utils';
 
-import '../scss/application.scss';
+class App extends Component {
+  renderLogin = () => loggedIn() ? <Redirect to="/game" /> : <Login />;
 
-function App() {
-  return <Game />;
+  render() {
+    return (
+      <ConnectedRouter history={history}>
+        <BrowserRouter>
+          <Switch>
+            <Route exact path="/" render={this.renderLogin} />
+            <AuthRoute path="/game" privateComponent={Game} />
```

Comment:
I would write just `component` since you already know it's an authenticated route due to your component's name (AuthRoute). Also I'd use this for publics routes too to make more generic. It's a component to check user authentication upon routes, not to just render private routes 🙌 

You could write something like this: 

```suggestion
            <AuthRoute exact path={ROUTES.DEFAULT_ROUTE} component={Login} redirectPath={ROUTES.GAME}/>
            <AuthRoute path={ROUTES.GAME} component={Game} />
```

Then in your AuthRoute component you can add the following: 

```js
function AuthRoute({ component: Component, redirectPath, isLogged, ...props }) {
  return isLogged ? <Route {...props} component={Component} /> : <Redirect to={redirectPath} />;
}
```

Remember to set a defaultProp for your redirectPath like so:

```js
defaultProps.AuthRoute = {
  redirectPath: ROUTES.DEFAULT_ROUTE
};
```